challenge-weather-app/api_keys.txt

```diff
@@ -0,0 +1,2 @@
+OpenWeather API Key: 6d0fbc92848b733471a1093b6ce30a2a
+Unsplash Access Key: ZN0G43FIimsZIYP8Duic-99-R8mkuVHJUSr9mVtThJY
```

Comment:
Just a quick heads up – it's important **NOT** to commit API keys or any sensitive information directly into the codebase. You can use `.env` file for them. https://www.codementor.io/@parthibakumarmurugesan/what-is-env-how-to-set-up-and-run-a-env-file-in-node-1pnyxw9yxj